src/NServiceBus.Msmq.AcceptanceTests/Distributor/When_worker_processes_a_message.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace NServiceBus.AcceptanceTests.Distributor
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+
+    public class When_worker_processes_a_message : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_also_send_a_ready_message()
+        {
+            var context = await Scenario.Define<DistributorEndpointTemplate.DistributorContext>()
```

Comment:
We have inconsistencies. Other test uses an inheritor of [`DistributorContext`](https://github.com/Particular/NServiceBus/pull/4329/files#diff-31425be79d15daf5aa4627be1818387cR39) which I prefer since it is more aligned with ATTs declaring the context locally

Also make `DistributorContext` abstract then?